js/components/MilkdropWindow/index.js

```diff
@@ -121,6 +129,14 @@ class MilkdropWindow extends React.Component {
       case 72: // H
         this._nextPreset(0);
         break;
+      case 145: // scroll lock
+        this.presetCycle = !this.presetCycle;
+        this._restartCycling();
+        break;
+      case 125: // F14 (scroll lock for OS X)
+        this.presetCycle = !this.presetCycle;
+        this._restartCycling();
+        break;
```

Comment:
One of the few times I think a [fall-through](https://en.wikipedia.org/wiki/Switch_statement#Fallthrough) would be preferable:


```
case 145: // scroll lock
case 125: // F14 (scroll lock for OS X)
  this.presetCycle = !this.presetCycle;
  this._restartCycling();
  break;
```

Reply:
oh yea, using a switch, might as well take advantage of it